### DISCUSSION TITLE
which ports are used???

### DISCUSSION THREAD

**Discussion by AmirMA-git** [hiddify/Hiddify-Manager#497] [Q&A سوالات رایج] [answered]
Hi there, I want to secure my cloud server so, I need to know which ports are used  in this platform.

I'm waiting for your answer, Thanks.

**Comment by zZedix** [comment, marked as answer]
the whole panel usually uses port 443 for proxy and 80 for ssl ! but there is a firewall option in the panel which you can turn on and it will secure the panel and server by itself !
